README.md

```diff
@@ -1,4 +1,158 @@
 # Analytics Platform Cran Proxy
+
 [![Docker Repository on Quay](https://quay.io/repository/mojanalytics/cran-proxy/status "Docker Repository on Quay")](https://quay.io/repository/mojanalytics/cran-proxy)
 
+- [Description](#sec-1)
+- [Background](#sec-2)
+- [Security](#sec-3)
+- [Usage](#sec-4)
+- [API](#sec-5)
+- [License](#sec-6)
+
+# Description<a id="sec-1"></a>
+
 A cran mirror that compiles binary packages for the rocker:verse environment.
+
+## Why<a id="sec-1-1"></a>
+
+CRAN servers usually provide binary versions of libraries for Windows and OSX
+but not for Linux. This is understandable because there's no such thing as
+*"Linux OS"*. Each distribution will have a set of C libraries, version of
+`libc` or a different implementation of `libc` altogether (`musl`).
```

Comment:
just to be really clear perhaps add something like: ", meaning you'd have to compile hundreds of binaries for each R package, to cope with the myriad of linux options."